### PR TITLE
Clone scanners will not beep while scanning if the clone pod is in use

### LINKED ISSUE #8219: clone pods make beeping constant noises
title, annoying as fuck

### PR DESCRIPTION
**Pull request by xTrainx**
:cl: xTrainx
fix: Clone scanners will not sound off if the clonepod is cloning
/:cl:

[why]: fixes #8219

### CODE REVIEW

**Review by JohnGinnane — DISMISSED** [review]
Looks good, did you test this?

**Comment by Carbonhell** [comment]
test it

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -456,7 +456,10 @@
 		return
 	if ((!mob_occupant.ckey) || (!mob_occupant.client))
 		scantemp = "<font class='bad'>Mental interface failure.</font>"
-		playsound(src, 'sound/machines/terminal_prompt_deny.ogg', 50, 0)
+		var/obj/machinery/clonepod/pod = GetAvailablePod()
+		if (!pod.occupant) //HIPPIE CODE -start- (PREVENTS SOUND WHILE THE POD IS IN USE)
```
Comment (Carbonhell) [inline code/game/machinery/computer/cloning.dm]: move the comment above, the code doesn't start here,no?

**Comment by JohnGinnane** [comment]
@xTrainx what's the story with this

**Review by Carbonhell — DISMISSED** [review]

**Review by Carbonhell — DISMISSED** [review]

**Comment by Carbonhell** [comment]
needs the edited tg file filepath to be added in changes.md (update your code before doing this)

**Review by Carbonhell — CHANGES_REQUESTED** [review]
update changes.md

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**